plugins/uefi-ssp-policy/fu-uefi-ssp-policy-plugin.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2024 Richard hughes <Richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include "fu-uefi-ssp-policy-plugin.h"
+
+struct _FuUefiSspPolicyPlugin {
+	FuPlugin parent_instance;
+};
+
+G_DEFINE_TYPE(FuUefiSspPolicyPlugin, fu_uefi_ssp_policy_plugin, FU_TYPE_PLUGIN)
+
+static gboolean
+fu_uefi_ssp_policy_plugin_bootmgr_found(void)
+{
+	/* FIXME: we only set this if bootmgr is not found */
```

Comment:
This really needs the new code in https://github.com/fwupd/fwupd/pull/7322 -- and then it's easy as checking for:

	<firmware gtype="FuEfiLoadOption">
	  <id>Windows Boot Manager</id>
	  <idx>0x2</idx>
	  <attrs>0x1</attrs>
	  <firmware gtype="FuEfiDevicePathList">
	    <firmware gtype="FuEfiHardDriveDevicePath">
	      <idx>0x4</idx>
	      <partition_number>0x1</partition_number>
	      <partition_start>0x800</partition_start>
	      <partition_size>0x32000</partition_size>
	      <partition_signature>c822f3ca-cfbc-4a61-b460-012f3d5be185</partition_signature>
	      <partition_format>guid-partition-table</partition_format>
	      <signature_type>guid</signature_type>
	    </firmware>
	    <firmware gtype="FuEfiFilePathDevicePath">
	      <idx>0x4</idx>
	      <name>/EFI/Microsoft/Boot/bootmgfw.efi</name>
	    </firmware>
	  </firmware>
	</firmware>

Reply:
Done!